src/components/UI/RestaurantsPage/Restaurants-Lists/Restaurant-Card/RestaurantCard.jsx

```diff
@@ -0,0 +1,73 @@
+import { Link } from "react-router-dom";
+import { useEffect,  useContext } from "react";
+import SkeletonCard from "./Skeleton-Cards/Skeleton-Card";
+
+import { RestroHome } from "../../../../../contexts/Restaurants/RestaurantsContext";
+const RestaurantCard = () => {
+  const { restaurants, productFetch, setRestaurants } = useContext(RestroHome);
+
+  useEffect(() => {
+    setTimeout(() => {
+      productFetch();
+    }, 2000);
+  }, [setRestaurants]);
+
+  return (
+    <>
+      {restaurants &&
+        restaurants.map((e) => {
```

Comment:
Suggestion don't write e here instead something meaningful, and destucture it here so dont have to repeatedly write e.